doc/man1/common/job-param-batch.rst

```diff
@@ -1,6 +1,7 @@
 .. option:: -n, --nslots=N
 
-   Set the number of slots requested. This parameter is required.
+   Set the number of slots requested. This parameter is required unless
+   :option:`--nodes` is specified.
```

Comment:
> All the gack about tasks doesn't apply and the --exclusive option doesn't have a short -x option.

I'm wondering if it is a bug that `flux alloc` doesn't support the `-x` short option? It seems a bit strange that it is support with `flux run` and `flux submit`, but not `flux alloc` and `flux batch`.

Reply:
Sure, I'll open one, then we can close it if we think it's better left as is.